WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingHeaderView.swift

```diff
@@ -27,25 +29,45 @@ class PrepublishingHeaderView: UIView, NibLoadable {
         backButtonView.layer.opacity = 0
         backButtonView.isHidden = true
         leadingConstraint.constant = Constants.leftRightInset
+        layoutIfNeeded()
     }
 
     func showBackButton() {
         backButtonView.layer.opacity = 1
         backButtonView.isHidden = false
         leadingConstraint.constant = 0
+        layoutIfNeeded()
     }
 
     @IBAction func backButtonTapped(_ sender: Any) {
         delegate?.backButtonTapped()
     }
 
+    // MARK: - Title
+
+    func setTitle(_ title: String?) {
+        titleLabel.text = title?.uppercased()
+    }
+
+    func showTitle() {
+        publishingToLabel.layer.opacity = 0
```

Comment:
Hm, why are we using multiple "title" labels? Why not just use a single label and adjust its title?

Reply:
Just because of the transition.

Changing the title right away is kinda harsh. However, the transition is quite quicker, so I don't have a strong opinion about that.

WDYT?

Reply:
Ah, makes sense. You can accomplish this with one label by doing the following:

```
func setTitle(_ title: String?) {
    UIView.transition(with: publishingToLabel as UIView,
                      duration: 0.2,
                      options: .transitionCrossDissolve,
                      animations: {
        self.publishingToLabel.text = title?.uppercased()
    })
}
```
Then you could reset the title to `Constants.title` if you pass nil?

Reply:
Done in 21e647262c51f494cbaeb266e4424b38fc21ada0